proxy_agent/src/provision.rs

```diff
@@ -146,57 +200,96 @@ pub async fn get_provision_status_wait(
             }
         }
 
-        return provision_status;
+        // wait timedout return as 'not finished'
+        return (false, String::new());
```

Comment:
should String::new() be "not finished".to_string() ?

Reply:
the first boolean parameter 'false' means not finished.